### PR TITLE
docs(hubspot): add particular properties reading example

### PR DESCRIPTION
**Pull request by IlyaFaer**
Into the Hubspot docs add a snippet, which shows how to extract particular properties of CRM objects.

Towards: https://github.com/dlt-hub/verified-sources/pull/311

### CODE REVIEW

**Review by rudolfix — COMMENTED** [review]
looks OK from my side! I'll let @AstrakhantsevaAA to decide if we should take the update here or in the verified sources?

the typical process was that we do it in verified sources and then the content is checked out / rewritten and posted here

File: docs/website/docs/dlt-ecosystem/verified-sources/hubspot.md [hunk 1]
```diff
@@ -172,7 +173,7 @@ def companies(
       api_key: str = dlt.secrets.value, include_history: bool = False
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/hubspot.md:173]: could you update this resource description? add new arguments "props" and "include_custom_props"

Comment (IlyaFaer) [reply docs/website/docs/dlt-ecosystem/verified-sources/hubspot.md]: Oh, yeah, done

**Comment by AstrakhantsevaAA** [comment]
there are some problem with tests, did you rebase on devel recently? if not, do it please, it can help

**Comment by AstrakhantsevaAA** [comment]
Actually you merge this PR from fork, these tests shouldn't run. check what's wrong, probably you really need to rebase

**Pull request closed without merging**